creditcoin-js/src/examples/ethereum.ts

```diff
@@ -64,11 +65,18 @@ const ethlessTransfer = async (
     return receipt;
 };
 
-export const ethConnection = async (providerRpcUrl = 'http://localhost:8545') => {
+export const ethConnection = async (
+    providerRpcUrl = 'http://localhost:8545',
+    minterWallet: Wallet | undefined = undefined,
```

Comment:
I've never seen this pattern before. what does it do?

Reply:
tells typescript that `minterWallet` can be of type `Wallet` or `undefined` b/c they are different types apparently and tsc isn't happy if only one is specified.